akri/Setup-Akri.ps1

```diff
@@ -0,0 +1,210 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+<#
+    .PARAMETER installAkri
+        installAkri boolean flag
+    
+    .PARAMETER uninstallAkri
+        uninstallAkri boolean flag
+
+    .PARAMETER kubeConfigFile
+        clusterName Path
+
+    .PARAMETER namespace
+        namespace
+#>
+param (
+    [Parameter()]

```

Comment:
Consider converting install/uninstall parameter to switches.

Reply:
i know ... i didn't like this either, but it is the existing style :(